draft-ietf-suit-report.md

```diff
@@ -214,9 +214,11 @@ SUIT_Report = {
   ? suit-report-nonce         => bstr,
   suit-report-records         => [ * SUIT_Record / system-property-claims ],
   suit-report-result          => true / {
+    suit-report-result-reason => SUIT_Report_Reasons,
     suit-report-result-code   => int, ; could condense to enum later
     suit-report-result-record => SUIT_Record,
```

Comment:
the cddl lists them in a different order (listing reason after record).  
Also the cddl deletes the "; could condense to enum later"


Be consistent so they match.